examples/models/image_classification/CNN.py

```diff
@@ -0,0 +1,201 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import base64
+import numpy as np
+import pickle
+import tensorflow as tf
+from tensorflow import keras
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import confusion_matrix
+import itertools
+from keras.layers import Dense, Dropout, Flatten, Conv2D, MaxPool2D
+from keras import layers
+from keras.layers.normalization import BatchNormalization
+from keras import models
+from keras.utils.np_utils import to_categorical
+from keras.optimizers import RMSprop
+from keras.preprocessing.image import ImageDataGenerator
+from keras.callbacks import ReduceLROnPlateau
+
+from rafiki.model import BaseModel, InvalidModelParamsException, test_model_class, \
+                        CategoricalKnob, FixedKnob, dataset_utils, logger
+from rafiki.constants import TaskType, ModelDependency
+
+class CNN(BaseModel):
+    '''
+    Implements CNN using keras for simple image classification
```

Comment:
Specifically which CNN is this? I believe we should rename this model to its exact CNN name. E.g. LeNet5, VGG are specific CNNs with a well-defined architecture

Reply:
this model's is not any specific well-defined architecture. Its architecture is of basic CNN, but layers are different from other well-defined ones. Believe that model developers can upload their own models on rafiki, so any kinds of models would be welcome as long as the result is acceptable?

Reply:
@nudles what do you think?

Reply:
i just changed to new model